stake-pool/program/tests/withdraw.rs

```diff
@@ -62,9 +62,9 @@ async fn setup() -> (
     .await
     .unwrap();
 
-    let tokens_to_burn = deposit_info.pool_tokens / 4;
+    let tokens_to_withdraw = deposit_info.pool_tokens / 4;
```

Comment:
much better, I never liked `tokens_to_burn` for some reason

Reply:
credit to @billythedummy